DESIGN_CONTRIBUTION.md

```diff
@@ -0,0 +1,119 @@
+# Contribution Process For Designers
+
+If you are designing a new component for Clarity or enhancing an existing component, please follow this step-by-step
+process in order to streamline the design, documentation and review phases of this process.
+We will work together with you on during each of these phases to guide you through our process.
+
+Using markdown the UX and Visual Design can be documented directly in a github issue. Please create a [Contribution proposal](https://github.com/vmware/clarity/issues/new?template=contribution-proposal.md),
+if there is not an existing issue related to the proposal. There are three phases a design goes through as it's refined and made ready for development.
+
+## UX Research
+
+UX research includes looking at the problems this component will solve. Identifying pre-existing solutions and the
+feature sets that should be included as part of a Clarity component.
+Usually answering the following questions will get you started
+
+### Getting Started
+
+Try answering the following questions to get started on researching and documenting the findings for conseration:
+
+* What problems does this component solve?
+* How will users interact with it?
+* What is the primary or base use case for this component?
+* Are there features that would enable less common but more enhanced use cases?
+
+When you write up the proposal, start by describing what the component does as well as anything known thats used today
+to solve the issue. Then list the use cases that can be paired with distinct visual designs.
+
+As the design is reviewed and use cases are refined some my be moved to a secondary list of **Future Use Cases**.
```

Comment:
Maybe define this more clearly by saying that we look to define what the MVP of the design is.